reference/mongodb/bson/persistable.xml

```diff
@@ -57,9 +57,11 @@
      </oointerface>
     </classsynopsisinfo>
 <!-- }}} -->
-    
+
+    <classsynopsisinfo role="comment">&Methods;</classsynopsisinfo>
+    <xi:include xpointer="xmlns(db=http://docbook.org/ns/docbook) xpointer(id('class.mongodb-bson-persistable')/db:refentry/db:refsect1[@role='description']/descendant::db:methodsynopsis[not(@role='procedural')])" />
+
     <classsynopsisinfo role="comment">&InheritedMethods;</classsynopsisinfo>
-    <xi:include xpointer="xmlns(db=http://docbook.org/ns/docbook) xpointer(id('class.mongodb-bson-serializable')/db:refentry/db:refsect1[@role='description']/descendant::db:methodsynopsis[not(@role='procedural')])" />
```

Comment:
I understand adding `Methods" but do we really need to remove list of inherited methods for Serializable? There's admittedly not a lot of guidance in our own docs for handling this, as "InheritedMethods" is only used elsewhere in our exception classes, none of which override methods.

If you consider [RecursiveTreeIterator](https://www.php.net/manual/en/class.recursivetreeiterator.php), all inherited methods are still listed below the actual methods and some of those _do_ override. I think that's OK, as each method links to a different definition. Showing both would also clearly communicate how we're narrowing the return type.

Reply:
Re-added the inherited method.